news/2025/2025-01-12-tournament-committee-applications-now-open.md

```diff
@@ -0,0 +1,74 @@
+- **Tournament Reviews**: Approving or denying badge support for tournaments and communicating any necessary changes to organisers.
+- **Handling Reports and Inquiries**: Investigating tournament-related issues reported by the community, assessing the validity of claims, and providing timely and well-considered responses to users. 
+- **Maintaining the [Official tournament support](/wiki/Tournaments/Official_support) wiki**: Keeping official rules and documentation up to date.
+- **Mentoring the Community**: Guiding newer members of the tournament scene towards making better decisions.
+- **Community Leadership**: Acting as a representative and leader within the tournament space.
+
+## Requirements for joining
+
+To join the TC, you must meet the following minimum criteria:
+
+- **Age**: You must be at least 18 years of age.
+- **Account Standing**: All applicants will undergo a security check. Any notable disciplinary actions within the last 12 months will disqualify a candidate.
+- **Language Proficiency**: Fluency in English is mandatory, as most reviewed content and all communications are in English.
+
+## What We Look for in Candidates
+
+In addition to meeting the requirements, successful candidates typically demonstrate the following qualities:
+
+- **Extensive Tournament Experience**: Has **2 years** of recent, meaningful involvement in tournaments as a player, staff member, or organiser. Community members will approach you with complex issues and expect you to provide sound guidance.
+- **Maturity and Professionalism**: Communicates respectfully and handles responsibilities with professionalism.
+- **Composure and Critical Thinking**: Displays a level-headed approach to analyzing multi-faceted situations while providing reasonable insight.
+- **Good Behaviour**: Possesses a **well-established reputation** within the community and tournaments.
+
+## Standout Qualities (Bonus Points!)
+
+We are particularly interested in candidates who:
+
+- Have a clear understanding of the tournament hosting process and badging criteria.
+- For staff members: Have hosted multiple badged tournaments or have experience as a tournament player.
+- For players: Have staffed multiple tournaments or hosted at least one badged tournament.
+- Consistently demonstrate a genuine desire to improve the tournament scene through their actions and advocacy.
+- Are fluent in languages other than English.
+
+# Decision Process
+
+All individuals under consideration will first be brought to open discussion, followed by an anonymous voting period. Following this vote, we will notify each candidate of the decision via email.
+
+For those selected, we will invite you to an optional 20-minute call with two randomly assigned TC members. This call serves as an opportunity to ask any questions you may have about the team, workload, or anything else before deciding to accept this role. We want you to feel confident and fully informed before making your decision to join the team. 
```

Comment:
so a job interview lol
certainly haven't heard any other osu! team do this

Reply:
This is specifically not labeled as an interview, it's optional and meant to be informative for anyone considering joining (who was selected). The TC workload is quite large.